packages/cron/src/jobs/goodbits.js

```diff
@@ -0,0 +1,137 @@
+import fetch, { Headers } from '@web-std/fetch'
+import path from 'path'
+import toml from 'toml'
+import ndjsonParser from 'ndjson-parse'
+
+/**
+ * @typedef {{ id: string, title: string }} Namespace
+ * @typedef {{ name: string, metadata: any }} Key
+ * @typedef {{ key: string, value: any, metadata?: any }} BulkWritePair
+ */
+
+const GOODBITS_SOURCES = [
+  'https://raw.githubusercontent.com/nftstorage/goodbits/fix/require-json-objects/list.ndjson',
+]
+
+const rootDir = path.dirname(path.dirname(import.meta.url))
+const wranglerConfigPath = path.join(
+  rootDir,
+  '../../edge-gateway/wrangler.toml'
+)
+
+export async function sync({ env }) {
+  const cfApiToken = mustGetEnv('CF_API_TOKEN')
+  const ghToken = mustGetEnv('GH_TOKEN')
+
+  const wranglerConfig = await getWranglerToml(wranglerConfigPath)
+  const wranglerEnvConfig = wranglerConfig.env[env]
+  if (!wranglerEnvConfig) {
+    throw new Error(`missing wrangler configuration for env: ${env}`)
+  }
+  console.log(`🧩 using wrangler config: ${wranglerConfigPath}`)
+
+  const cfAccountId = wranglerEnvConfig.account_id
+  if (!cfAccountId) {
+    throw new Error(`missing Cloudflare account_id in env: ${env}`)
+  }
+  console.log(`🏕 using env: ${env} (${cfAccountId})`)
+
+  const kvNamespaces = wranglerEnvConfig.kv_namespaces || []
+  const goodbitsListKv = kvNamespaces.find(
+    (kv) => kv.binding === 'GOODBITSLIST'
```

Comment:
nit: other uses of `GOODBITS` in this PR use an underscore separator e.g. `GOODBITS_SOURCES`

Reply:
yeah, I went this naming to be consistent with KV for `DENYLIST` (no underscore there sadly)